### DISCUSSION TITLE
production application giving the "not whitelested" error

### DISCUSSION THREAD

**Discussion by KetulCodiste** [Web3Auth/Web3Auth#1195] [Help]
I created a project and received the clientId from the web3auth dash board plug and play. I am able to login to the web3auth from localhost, but after deploying the website to the vercel, it is giving me following error:

```
Error: could not validate redirect, please whitelist your domain: https://********.vercel.app for provided clientId BCp-*******sE at https://dashboard.web3auth.io.
    Also, this project is on mainnet network. Please ensure the the used Client ID belongs to this network.
```

I have added my domain in the dashboard for whitelisting and I am using the testnet. Still the production website is giving the error of not whitelisting the domain and even though I have used the testnet, it is saying that the project is on mainnet.

**Comment by shahbaz17** [comment]
Hey @KetulCodiste 

Which Web3Auth SDK are you using? And what's the version of that?

Also, please share the code snippet of web3auth initialization and login.

**Comment by KetulCodiste** [comment]
https://github.com/orgs/Web3Auth/discussions/1115
I found the solution here...
